std_msgs/Header header

bool main_cab_active
bool driving
bool driving_forward
bool braking_normal
bool braking_emergency
bool braking_rail
bool braking_forced
bool not_spring_break
bool switch_control_left
bool switch_control_right
bool sanding
bool doors_open
bool pantograph_up
bool indicator_right
bool indicator_left
bool hazard_lights
bool warning_bell
float32 driver_setpoint
float32 velocity            #in m/s
float32 velocity_kph        #in km/h